coalib/bears/requirements/PipRequirement.py

```diff
@@ -33,11 +31,11 @@ def install_command(self):
         """
         Creates the installation command for the instance of the class.
 
-        :param return: A string with the installation command.
+        :param return: A list with the installation command parameters.
         """
-        result = "{} install {}".format(self.MANAGER, self.package)
-        if self.version:
-            result += "=={}".format(self.version)
+        result = [sys.executable, '-m pip', 'install',
```

Comment:
ok I'm super sorry, I told you some nonsense! Apparently you have to split the arguments, i.e. `[sys.executable, '-m', 'pip', ...`

I don't fully understand this yet, I think you basically give the `sys.args` list in which `-m` and `something` could be different things. If you'd invoke a program like `program -u "hell yeah"` you could also do `subprocess.call(['program', '-u', 'hell yeah'])`.